namd/tests/library/030_neuralnetwork/test.in

```diff
@@ -0,0 +1,107 @@
+colvarsTrajFrequency    1
```

Comment:
This configuration is quite portable, but it is run by a job script to NAMD (2.x branch) and does not test a restarting job. This means that in the current setup it won't be tested for other backends.

Can you make an input based on decaalanine, so that we can test the other backends as well? I can propagate it if you like but would need updated .txt files.

Reply:
I change the input for testing to deca-alanine.